### ISSUE TITLE
polyfill-corejs3 inject wrong Number.isNaN polyfill

### ISSUE THREAD

**Issue by fangbinwei** [babel/babel-polyfills#69]
https://github.com/babel/babel-polyfills/blob/9f43e20e71005cf00963c2653960a393c368c224/packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.js#L269-L270

maybe should be 
```js
isNaN: define("number/is-nan", ["es.number.is-nan"]), 
 isSafeInteger: define("number/is-safe-integer", ["es.number.is-safe-integer"])
```

I can provide a reproducible repo if needed

related to https://github.com/vuejs/vue-cli/issues/6324

**Comment by sodatea** [comment]
Same case for `MAX_SAFE_INTEGER` and `MIN_SAFE_INTEGER`:
https://github.com/babel/babel-polyfills/blob/9f43e20e71005cf00963c2653960a393c368c224/packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.js#L260-L265

**Comment by nicolo-ribaudo** [comment]
Whops sorry for this, would you like to open a PR?

**Comment by wangziling** [comment]
And maybe, here:
https://github.com/babel/babel-polyfills/blob/9f43e20e71005cf00963c2653960a393c368c224/packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.js#L237-L238

**Comment by wangziling** [comment]
Add a pr. #70

**Comment by nicolo-ribaudo** [comment]
Fixed in `babel-plugin-polyfill-corejs3@0.1.7`. Thanks @wangziling!
